requirements/default.txt

```diff
@@ -368,9 +370,9 @@ idna==2.10 \
     # via
     #   -r requirements/default.in
     #   requests
-jinja2==2.11.3 \
-    --hash=sha256:03e47ad063331dd6a3f04a43eddca8a966a26ba0c5b7207a9a9e4e08f1b29419 \
-    --hash=sha256:a6d58433de0ae800347cab1fa3043cebbabe8baa9d29e668f1c768cb87a333c6
+jinja2==3.0.1 \
```

Comment:
Doesn't seem to be anything breaking here, but a couple of deprecations which affect us:

https://jinja.palletsprojects.com/en/3.0.x/changes/#version-3-0-0

> Deprecate the autoescape and with extensions, they are built-in to the compiler.

> pass_context replaces contextfunction and contextfilter.